bin/exec.ml

```diff
@@ -77,22 +77,28 @@ module Command_to_exec = struct
 
   (* Helper function to spawn a new process running a command in an
      environment, returning the new process' pid *)
-  let spawn_process path ~args ~env =
+  let spawn_process (common : Common.t) path ~args ~env =
```

Comment:
Same thing here. A function taking `common` as an argument is essentially function that takes every single configuration value and command line flag. It should just take `Common.root` as that' the only thing it needs from this record.

 I can fix this one